src/ripple/core/JobQueue.h

```diff
@@ -108,7 +108,7 @@ class JobQueue
     using JobFunction = std::function <void(Job&)>;
 
     JobQueue (beast::insight::Collector::ptr const& collector,
-        Stoppable& parent, beast::Journal journal, Logs& logs);
+        Stoppable& parent, beast::Journal journal, Logs& logs, bool validator);
```

Comment:
Nitpick: conceptually, I don't like passing this information into `JobQueue`. No change necessary at this time.